build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.88.1-alpha1'
+    gutenbergMobileVersion = '5485-581dfd445660821663474c13f9b55c3057aaf182'
```

Comment:
Friendly reminder: we'll need to update this to the Gutenberg release tag once it is published.